js/ord-schema/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ord-schema",
-  "version": "0.3.94",
+  "version": "0.3.95",
```

Comment:
you should not update the version manually; bump2version handles this: https://github.com/open-reaction-database/ord-schema/blob/main/.bumpversion.cfg

Reply:
Okay, will revert the changes